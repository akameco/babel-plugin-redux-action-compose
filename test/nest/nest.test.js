// @flow
import path from 'path'
import { transformFileSync } from 'babel-core'
import plugin from '../../src'

test('nest snapshot', () => {
  const input = path.join(__dirname, 'actions.js')

  const { code } = transformFileSync(input, {
    babelrc: false,
    plugins: [
      [plugin, { input: path.join(__dirname, 'other/sub/actions.js') }],
    ],
  })
  expect(code).toMatchSnapshot()
})
