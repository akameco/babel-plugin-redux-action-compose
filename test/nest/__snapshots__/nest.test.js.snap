// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`nest snapshot 1`] = `
"
// @flow
import type { Action as Other2Action } from './other2/actions';
import type { Action as Other3Action } from './other3/actions';

import type { Action as SubAction } from './other/sub/actions';

export type Action = Other2Action | Other3Action | SubAction;"
`;
